### PR TITLE
[Student][E2E][MBL-13006]: Initial attempt at Quizzes E2E test

### PR DESCRIPTION
**Pull request by joehoag**
Also includes some accessibility fixes.

Please LMK if you can think of any useful ways to beef up this E2E test.

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1005742&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

File: automation/canvas_espresso/src/main/kotlin/com/instructure/canvas/espresso/CanvasTest.kt [hunk 1]
```diff
@@ -7,13 +7,18 @@ import android.view.View
 import android.webkit.WebView
```
Comment (StephenBrough) [inline automation/canvas_espresso/src/main/kotlin/com/instructure/canvas/espresso/CanvasTest.kt]: Probably want a license in this file

**Comment by joehoag** [comment]
@StephenBrough : Thanks for the feedback.  I'll push a commit addressing your suggestions (copyright notice + "beefing up") before merging these fixes.

**Comment by joehoag** [comment]
Latest commit adds the copyright header.  It also starts the quiz via the Assignments tab and resumes/completes the quiz via the Quizzes tab.

**Pull request merged**